app/src/ble.c

```diff
@@ -72,10 +76,25 @@ static void security_changed(struct bt_conn *conn, bt_security_t level,
     }
 }
 
+static bool le_param_req(struct bt_conn *conn, struct bt_le_conn_param *param) {
```

Comment:
Since this function is unused unless this is a peripheral, the whole function should be wrapped in an `#if`